pol-core/pol/multi/boat.cpp

```diff
@@ -1249,24 +1241,24 @@ void UBoat::do_tellmoves()
 }
 
 // dave 3/26/3 added
-bool UBoat::move_xy( unsigned short newx, unsigned short newy, int flags, Realms::Realm* oldrealm )
+bool UBoat::move_xy( const Core::Pos2d& newp, int flags,
+                     Realms::Realm* oldrealm )  // Todo Pos refactor oldrealm??
 {
   bool result;
   BoatMoveGuard registerguard( this );
-
-  if ( ( flags & Core::MOVEITEM_FORCELOCATION ) ||
-       navigable( multidef(), newx, newy, z(), realm() ) )
+  Core::Pos4d newpos{ newp, z(), realm() };
+  if ( ( flags & Core::MOVEITEM_FORCELOCATION ) || navigable( multidef(), newpos ) )
   {
     BoatContext bc( *this );
 
     set_dirty();
-    move_multi_in_world( x(), y(), newx, newy, this, oldrealm );
+    move_multi_in_world( x(), y(), newpos.x(), newpos.y(), this, oldrealm );
```

Comment:
did move_multi_in_world and move_travellers get forgotten or you have a reason to leave them with separate x,y inputs?

Reply:
I stopped at this point for this PR :)